src/app/components/pages/Login/index.tsx

```diff
@@ -1,44 +1,72 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { API_URL, PAGES } from '../../../utils/contants';
+
 import './index.css';
+import background from '../../../assets/loginBackground.svg';
+
 export interface LoginProps {
   setActivePage: React.Dispatch<React.SetStateAction<PAGES>>;
 }
+
+function generateRandomToken() {
+  const randomValues = new Uint8Array(128);
```

Comment:
this is 128 bytes which is too big. We need 128 bits(16 bytes).

Reply:
I searched around and looks like better alternative is to directly use
`crypto.randomUUID()`
It returns uuid in uuid format which is easier to recognize also.
'554b4597-1cc6-451d-ae67-e12234a10805'

Reply:
I am not 100% sure but the documentation says only ```getRandomValues``` method is available in insecure context. Other methods are hence not available. 

I tested the ```randomUUID()``` method, but it doesn't seem to be working. (undefined function)
![Screenshot from 2023-12-25 22-14-37](https://github.com/croma-app/huehive-figma-plugin/assets/133551284/89d38c88-150b-4f27-b2bb-70202aaef2c7)

Link to the doc:
#https://developer.mozilla.org/en-US/docs/Web/API/crypto_property

Reply:
yeah looks like this is only available in https or localhost and figma plugin might be using file 
We can use the function from here I think 
https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid